docs/reference/transactions.txt

```diff
@@ -145,6 +166,14 @@ session if one is in progress:
   # ok
   c2.database.drop
 
+Handling Errors
+---------------
+
+If a command inside the transaction fails with, the transaction may be aborted
```

Comment:
"If a command inside the transaction fails with," .... fails with what?

Reply:
Fixed, thanks!